src/components/organisms/WithNavigationBar/WithNavigationBar.scss

```diff
@@ -10,3 +11,29 @@
   overflow: hidden;
   flex-flow: column;
 }
+
+.footer-wrapper {
+  display: flex;
+  align-items: center;
+  border-top: 1px solid rgba(255, 255, 255, 0.2);
+  position: fixed;
+  height: 66px;
+  bottom: 0;
+  left: 0;
+  z-index: 300;
```

Comment:
q: how did you arrive at `300` for `z-index` here

Reply:
As mentioned on another PR, we should extract all z-index 'layers' to our variables, and then we can easily see what they all are, and 'adjust' any layering issues in a single place.

Reply:
a suggestion of z-index layering

https://github.com/IvanBuljovcic/React-Boilerplate/blob/master/src/css/variables/zindex.yml

Reply:
I like the concept/pattern of it, though would prefer to use some more semantically relevant names when we do it.

Reply:
^extracted task for this to https://github.com/sparkletown/internal-sparkle-issues/issues/6

Reply:
@denisdimitrov will review/revise the z-index stuff in a followup PR